app.css

```diff
@@ -23,175 +23,228 @@
+	font-family: sans-serif;
+	font-size: 9pt;
+	border-bottom: 1px solid rgb(199, 198, 198);
+	padding: 2px 0px;
+	max-width: 240px;
 }
 
 .featureInfo label{
-    margin-right: 6px;
-    font-family: sans-serif;
+	margin-right: 6px;
+	font-family: sans-serif;
 }
 
 .featureInfo img, .leaflet-popup-content img{
-    max-height:  200px;
+	max-height:  200px;
 }
 
 .leaflet-popup-content {
-    max-height: 290px;
-    overflow-y: auto;
+	max-height: 290px;
+	overflow-y: auto;
 }
 
 .featureCompliance{
-    text-align:center;
+	text-align:center;
 }
 
 .howTo{
-    font-size: 10pt;
+	font-size: 10pt;
 }
 
 input[app]{
-    margin: 2px 0px 5px 0px;
+	margin: 2px 0px 5px 0px;
 }
 
 @keyframes color {
-    0% {
-      background-color: white;
-    }
-    50% {
-      background-color: darkgrey;
-    }
-    100% {
-      background-color: white;
-    }
+	0% {
+		background-color: white;
+	}
+	50% {
+		background-color: darkgrey;
+	}
+	100% {
+		background-color: white;
+	}
 }
 footer{
-  bottom : 0;
-  background-color : #FFF;
-  z-index : 1;
-  flex-flow:  wrap;
+	bottom : 0;
+	background-color : #FFF;
+	z-index : 1;
+	flex-flow:  wrap;
 }
 footer div{
-    display:  inline-flex;
+	display:  inline-flex;
 }
 header {
-    background: #222;
-    width: 100%;
-    padding: 1rem;
-    position: relative;
-    box-shadow: 0 0 3rem #001e3c;
-    margin-bottom: 3rem;
+	background: #222;
+	width: 100%;
+	padding: 1rem;
+	position: relative;
+	box-shadow: 0 0 3rem #001e3c;
+	margin-bottom: 3rem;
 }
 header small {
-    position: absolute;
-    bottom: 1rem;
-    line-height: 1;
-    margin-left: 17.1%;
-    color: #dfdfdf;
+	position: absolute;
+	bottom: 1rem;
+	line-height: 1;
+	margin-left: 17.1%;
+	color: #dfdfdf;
 }
 header h1 {
-    color: #a3d1ff;
-    letter-spacing: .55rem;
-    position: relative;
-    font-weight: 300;
-    padding-left: 1em;
-    font-size:  75%;
+	color: #a3d1ff;
+	letter-spacing: .55rem;
+	position: relative;
+	font-weight: 300;
+	padding-left: 1em;
+	font-size:  75%;
 }
 header img{
-    max-width: 69%;
-    padding-left: 6px;
-    max-height:  8em;
+	max-width: 69%;
+	padding-left: 6px;
+	max-height:  8em;
 }
 .card header{
-    background-color: white;
-    box-shadow: none;
-    margin-bottom: 1em;
-    padding: 0px;
+	background-color: white;
+	box-shadow: none;
+	margin-bottom: 1em;
+	padding: 0px;
 }
 
 .regular-header-text{
-    top:  1em;
-    position:  relative;
-    font-size:  14pt;
+	top:  1em;
+	position:  relative;
+	font-size:  14pt;
 }
 
 .small-header-text{
-    position:  relative;
-    font-size:  9pt;
+	position:  relative;
+	font-size:  9pt;
+}
+
+.tooltip{
+	position: relative !important;
+	display: flex;
+}
+
+.tooltip .tooltiptext{
+	visibility: hidden;
+	width:120px;
+	background-color: black;
+	color: #fff;
+	text-align: center;
+	/*font-size: 30px;*/
+	border-radius: 6px;
+	padding: 5px 0;
+	position: absolute;
+	z-index: 1;
+	bottom: 110%;
+	left: 50%;
+	margin-left: -60px;
+
+}
+
+span{
```

Comment:
This is too global because it affects all `<span>` elements.  It is going to have side effects outside of your scope.  You'll need a way to contain this to just those elements which you intend to affect.  I think this is why the '+' and '-' symbols on the map zoom buttons are too small now.